### PR TITLE
Task/bin346 implement vision mision and values in about us page

### PR DESCRIPTION
**Pull request by GustavoUT22**
# Implement What We Are Section

## Summary
The WhatWeAre section highlights Binland’s mission, vision, and values. It uses a reusable WhatWeAreCard component styled with styled-components and organized with WhatWeAre.data for easy content management. This modular approach ensures a consistent and scalable user experience.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Refactor
- [X] Styling
- [ ] Testing
- [ ] Chore
- [ ] Other: ...

## Checklist:

These items must be completed before the code review

- [ ] Check that branch is pointing to production environment
- [X] Remove console.logs
- [X] Check imports format
- [X] Check variables destructure
- [ ] Check docs
- [X] Review the entire pull request yourself before sending it to the reviewer
- [ ] Check task requirements on Monday or Sprint Planning sheet

## Screenshots or Gifs
image
image

### CODE REVIEW

File: src/components/Cards/WhatWeAreCard/WhatWeAreCard.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+const WhatWeAreCard: React.FC<IWhatWeAre> = (props) => {
+  const intl = useIntlMessages()
+  return (
+    <CardWrapper>
+      <div className="info">
+        <H3 $weight="bold">{intl(props.title).toUpperCase()}</H3>
+        <p className="mision__description">{intl(props.description)}</p>
+      </div>
+      <WhatWeAreImg
```
Comment (mardecera) [inline src/components/Cards/WhatWeAreCard/WhatWeAreCard.tsx:14]: You're not using lazy loading for the images.

**Review by mardecera — CHANGES_REQUESTED** [review]
"When naming images, describe them in the shortest way possible. If you include where they are used in the name, it will lose meaning when used elsewhere.

`Example: 'Image of a blue apple' -> name: blue-apple"`

File: src/components/Cards/WhatWeAreCard/WhatWeAreCard.tsx (hunk 1 above)
Comment (GustavoUT22) [reply src/components/Cards/WhatWeAreCard/WhatWeAreCard.tsx:14]: done!

**Review by mardecera — APPROVED** [review]
Nice

**Review by CrhistianT7 — APPROVED** [review]

**Pull request merged**